duckduckgo/Android
# Delete no longer relevant legacy migration code

Task/Issue URL: https://app.asana.com/0/361428290920652/1108209308539149

**Description**:
Delete legacy db migration code

**Steps to test this PR**:
1. Run the app
1. Make sure nothing goes 💥

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
